python/arcticdb/version_store/helper.py

```diff
@@ -333,10 +333,12 @@ def get_azure_proto(
     container_name,
     endpoint,
     with_prefix: Optional[bool] = True,
-    ca_cert_path: Optional[str] = None,
+    ca_cert_path: Optional[str] = "",
 ):
     env = cfg.env_by_id[env_name]
     azure = AzureConfig()
+    if not container_name:
```

Comment:
It does not work but I have never realised empty container should be acceptable. I need to dig deeper to confirm who has posed the limitation (us or the sdk)

Reply:
Currently ArcticDB does not support azure root container. It is because azure url is in the format of `[ENDPOINT]/[CONTAINER]/[KEY]`. To access root container, not by "$root`, the url will look like this `[ENDPOINT]/[KEY]`. However, the key, currently, could look like this `_arctic_cfg/cref/*sUt*p...`. As `/` is used in the key generation, everything between the endpoint and the first slash will be treated as the container. 

A straight forward fix (maybe hacky) is setting container default parameter to be `$root`. Other than that will requires some work

Reply:
I think this is fine. We can document in the website how to use the root container at some point.